crates/testing/tests/tests_1/test_with_failures_2.rs

```diff
@@ -52,3 +52,39 @@ cross_tests!(
         metadata
     }
 );
+
+#[cfg(not(feature = "dependency-tasks"))]
+cross_tests!(
+    TestName: test_with_double_leader_failures,
+    Impls: [MemoryImpl, Libp2pImpl, PushCdnImpl],
+    // this lets a node be a leader two views in a row
+    Types: [TestConsecutiveLeaderTypes],
+    Ignore: false,
+    Metadata: {
```

Comment:
We have a view sync test task that will actually check to see if we hit view sync (or enforce that nodes don't enter view sync), maybe you can add that to this test to show that nodes must hit view sync

Reply:
ok, in test im setting view sync properties threshold now